### DISCUSSION TITLE
Is the canvas transparent?

### DISCUSSION THREAD

**Discussion by tomcrane** [digirati-co-uk/iiif-canvas-panel#98] [Q&A]
I think it is, by default.

You can set the background colour of CP, which would give you this, if set to grey:

![image](https://user-images.githubusercontent.com/1443575/140778950-4f49cdec-9e0c-469d-85f4-ba168ffbff1b.png)

Otherwise (and assuming a grey field image hasn't been placed on the canvas!) the web page will "show through" the non-image areas.

**Comment by stephenwf** [comment]
Yes that's right, the background colour can be customised. But maybe the "canvas" background would be different to the "world". I'm not sure how that could be specified in the data, or if it should only be a decision from the viewer.
